book/website/collaboration/github/github-roadmapping.md

```diff
@@ -1,10 +1,10 @@
-(rr-github-roadmapping)=
+(cl-github-roadmapping)=
 # Roadmapping
 
-If you plan a larger piece of work, it is very useful to have an outline for the work you need to do and share it with potential contributors.
-Your roadmap covers your goal and vision and should include a timeline for tasks that need to be completed, thus helping anyone new to your project to develop an understanding of what is currently happening on the project and what's coming next.
+If you plan a large piece of work, it will be good to produce an outline of the work you need to do and share it with potential contributors.
+Your roadmap should cover your goal and vision and include a timeline for tasks that need to be completed, thus helping new ones to your project to get an understanding of what is currently happening on the project and what's coming next.
```

Comment:
I would replace "new ones" with "new contributors" so that it's explicit.

Reply:
Thanks @inwaves , will do that.